bciers/apps/registration/app/components/profile/ProfileForm.tsx

```diff
@@ -0,0 +1,152 @@
+"use client";
+import { useState } from "react";
+import { Alert } from "@mui/material";
+import { actionHandler } from "@bciers/actions";
+import FormBase from "@bciers/components/form/FormBase";
+import { Button } from "@mui/material";
+import { useSession } from "next-auth/react";
+import { useRouter } from "next/navigation";
+import { RJSFSchema } from "@rjsf/utils";
+import FieldTemplate from "@bciers/components/form/fields/FieldTemplate";
+
+export interface UserProfileFormData {
+  first_name: string;
+  last_name: string;
+  position_title: string;
+  email: string;
+  phone_number: string;
+  app_role?: { role_name: string };
+}
+
+export interface UserProfilePartialFormData {
+  first_name?: string;
+  last_name?: string;
+  position_title?: string;
+  email?: string;
+  phone_number?: string;
+  app_role?: { role_name: string };
+}
```

Comment:
I don't see why we need the `UserProfileFormData` and `UserProfilePartialFormData` interfaces defined again here when they're already defined in bciers/libs/shared/types/src/form/formData.ts ?